src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  margin: 0;
+  padding: 0;
+
+  --hoverlink: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
```

Comment:
better add for the curent element or list, not for all body

May be you will have more blocks in body in the future so you will have to change existing styles